src/bundle/Resources/config/ezplatform_default_settings.yaml

```diff
@@ -74,5 +74,8 @@ parameters:
     ezsettings.default.content_tree_module.contextual_tree_root_location_ids: []
 
     # Autosave
+    ezsettings.default.autosave.enabled: true
```

Comment:
Is it really necessary? Autosave should only be enabled for `admin_group`, not for every site access.

Reply:
Yes, but we do not have system that can enable/disable settings based on SA, and in current state no value for those two in different SA than admin causes not found exception.

Reply:
Makes sense, but change it to `false` (disable by default), otherwise it may result in unexpected behaviour in case of UGC.